src/utils/generateScriptForSSR.js

```diff
@@ -0,0 +1,31 @@
+// @flow
+
+import themeWithCssVariables from "./themeWithCssVariables";
+import type { PrismTheme } from "../types";
+
+const generateScriptForSSR = (
+  themes: PrismTheme[],
+  getThemeIdFuncStr: string
+): string =>
+  `
+const themeId = (${getThemeIdFuncStr})();
+
+const root = document.documentElement;
+
+${themes
+  .map(
+    (theme) =>
+      `if (themeId === '${theme.id || ""}') {
+  ${Object.entries(themeWithCssVariables(theme).variables)
+    .map(
+      ([key, value]) =>
+        // $FlowFixMe
+        `root.style.setProperty('${key}', '${value || ""}');`
+    )
+    .join("\n" + " ".repeat(2))}
+}`
+  )
+  .join("\n\n")}
+`.trim();
```

Comment:
Would it make sense to wrap this in a `try/catch` so that if anything happens here, we don't have unexpected side-effects? (Like, I'd hate for something to _boop_ here and then crash the whole page or something – but maybe that's not a realistic scenario.)

Reply:
Yeah, a `try/catch` wouldn't hurt! I'll add it.

Reply:
Done.